genesyscloud/flow_loglevel/resource_genesyscloud_flow_loglevel_schema.go

```diff
@@ -0,0 +1,101 @@
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+			"execution_input_outputs": {
+				Description: "Whether to report input setting values and output data values for individual execution items above.  For example, if you have FlowExecutionInputOutputs and a Call Data Action ran in a flow, if FlowExecutionItems was enabled you'd see the fact a Call Data Action ran and the output path it took but nothing about which Data Action it ran, the input data sent to it at flow runtime and the data returned from it.  If you enable this characteristic, execution data will contain this additional detail.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+			"execution_items": {
+				Description: "Whether to report execution data about individual actions, menus, states, tasks, etc. etc. that ran during execution of the flow.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+			"names": {
+				Description: "This characteristic specifies whether or not name information should be emitted in execution data such as action, task, state or even the flow name itself.  Names are very handy from a readability standpoint but they do take up additional space in flow execution data instances.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+			"variables": {
+				Description: "Whether to report assignment of values to variables in flow execution data. It's important to remember there is a difference between variable value assignments and output data from an action.  If you have a Call Digital Bot flow action in an Inbound Message flow and there is no variable bound to the Exit Reason output but FlowExecutionInputOutputs is enabled, you will still be able to see the exit reason from the digital bot flow in execution data even though it is not bound to a variable.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+			},
+		},
+	}
+	return &schema.Resource{
+		Description: `Genesys Cloud flow log level`,
+
+		CreateContext: provider.CreateWithPooledClient(createFlowLogLevel),
+		ReadContext:   provider.ReadWithPooledClient(readFlowLogLevel),
+		UpdateContext: provider.UpdateWithPooledClient(updateFlowLogLevel),
+		DeleteContext: provider.DeleteWithPooledClient(deleteFlowLogLevel),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			"flow_id": {
+				Description: "The flowId for this characteristics set",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"flow_log_level": {
+				Description: "The logLevel for this characteristics set",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"flow_characteristics": {
+				Description: "Shows what characteristics are enabled for this log level",
+				Type:        schema.TypeList,
+				Computed:    true,
+				Elem:        flowCharacteristics,
```

Comment:
I would add `MaxItems: 1`  since the user shouldn't define more than one `flow_characteristics` block